### PR TITLE
feat: integrate dockerfile to create frontend images

### PR DESCRIPTION
**Pull request by anversa-pro**
## Issue ticket number and link
https://trello.com/c/qAnSqSN7

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [x] :sparkles: Feature
- [ ] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [x] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

-

## What is the new behavior?

- The dockerfile has been configured to build frontend images.
-  A make command has been created to facilitate image building.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+COPY yarn.lock ./
+
+RUN yarn cache clean && yarn install
+
+# Bundle app source
+COPY . .
+
+EXPOSE 3000
+# Silent start because we want to have our log format as the first log
+CMD ["yarn", "start"]
```
Comment (FelipeLosada1996) [inline Dockerfile:17]: Aunque funciona, puede no ser lo más eficiente, sobretodo combinado el lock, debido al donde se compile. Dejo otras posibilidades https://dev.to/otomato_io/how-to-optimize-production-docker-images-running-nodejs-with-yarn-504b

Comment (FelipeLosada1996) [reply Dockerfile:17]: Toca revisar que ya no es correrlo en local, sino en un servicio del cloud deseado

Comment (jose-sal-y-rosas) [reply Dockerfile:17]: Please @FelipeLosada1996 two suggestions:
1. Use english
2. We are definitely going to use just the build folder to publish the application using nginx

**Review by jose-sal-y-rosas — COMMENTED** [review]
@anversa-pro please let me know if you need some help

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Review by FelipeLosada1996 — APPROVED** [review]

**Pull request merged**